include/SPTPersistentCache/SPTPersistentCacheOptions.h

```diff
@@ -114,6 +114,12 @@ FOUNDATION_EXPORT const NSUInteger SPTPersistentCacheMinimumExpirationLimit;
  */
 @property (nonatomic, copy) NSString *cachePath;
 
+/**
+ Excludes the cache directory from backup.
+ @discussion Some users may wish to have the cache directory excluded from backup.
```

Comment:
@aodhol In order to avoid any confusion, would be nice to mention the default value here maybe.